plugins/bundle/plugin.go

```diff
@@ -0,0 +1,282 @@
+// Copyright 2018 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+// Package bundle implements bundle downloading.
+package bundle
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/bundle"
+	"github.com/open-policy-agent/opa/plugins"
+	"github.com/open-policy-agent/opa/storage"
+	"github.com/open-policy-agent/opa/util"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// min amount of time to wait following a failure
+	minRetryDelay = time.Millisecond * 100
```

Comment:
Can't this be configurable as well ?

Reply:
Can definitely make this configurable. I'll leave this for a follow up PR (or if someone else wants/needs it, they can do so.)